components/vf-button/vf-button.config.yml

```diff
@@ -22,30 +23,6 @@ variants:
       text: a small button
       theme: primary
       size: sm
-  - name: regular
```

Comment:
I think removing them will break templates explicitly using that variant (most likely in WP). To avoid a 2.0.0 release and make Szymon's life easier, a `hidden: true` should achieve the same aims in your PR?

Reply:
👍